extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -385,9 +385,45 @@ describes.realWin(
             'https://server-test-3/':
               '{"consentRequired": true, "consentStateValue": "unknown"}',
             'https://geo-override-check2/': '{"consentRequired": true}',
+            'https://gdpr-applies/':
+              '{"consentRequired": true, "gdprApplies": false}',
+            'https://gdpr-applies-2/': '{"consentRequired": true}',
           };
         });
 
+        describe('gdprApplies value', () => {
+          it('uses given value', async () => {
+            const inlineConfig = {
+              'consentInstanceId': 'abc',
+              'consentRequired': 'remote',
```

Comment:
Please also add a test to test the local `consentRequired: false` or `consentRequired: true` as well.

Reply:
Actually it will also work if you decide to limit the `gdprApplies` with `checkConsentHref`. But in that case, maybe we should even expose the `getGdprAppliesInfo` API. That it should only be retrieved with consent string.

Reply:
> Actually it will also work if you decide to limit the gdprApplies with checkConsentHref. 

Yes this is currently the case. In the spec, we said that it would default to `consentRequired` and I interpreted that to mean that we check `checkConsentHref` first then use `consentRequired` if it's not given.

> That it should only be retrieved with consent string

So are you saying that we should remove `getGdprAppliesInfo` and `getConsentPolicyInfo` from `src/consent` and create a general API (ie: `getTcfV2Info()`)?

Reply:
Ha the `consentRequired` defined in different places is confusing. What about the `consentRequired` in inline config?

Reply:
Discussed offline: will save consolidation of these APIs for later work (by June), so that we can unblock ads teams that are needing the `gdprApplies` data.